frontend/src/shared/lib/trimAllObjectValues.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Применяет trim ко всем полям объекта
+ * @param object объект для изменения
+ */
+export const trimAllObjectValues = (object: object) => {
+    return Object.fromEntries(
+        Object.entries(object).map(([key, value]) => [key, value!.trim()]),
+    );
```

Comment:
Можно сделать еще более общей, чтобы парсить все обьекты:
```suggestion
    return Object.fromEntries(
        Object.entries(object).map(([key, value]) => 
            [key,  typeof value === 'string' ? value!.trim() : value] 
        ),
    );
```